config/server-vars.yml

```diff
@@ -76,6 +76,7 @@ EDXAPP_FEATURES:
   ENABLE_SYSADMIN_DASHBOARD: true
```

Comment:
I noticed you're merging this into oxa/master.euc -- that's awesome. is the "master" branch dogwood? if so, i'm thinking we rename that branch oxa/master in keeping with the pattern we use elsewhere. i'm not going to block your checkin based on this, but I do want to the team do have a discussion about it soon